README.md

```diff
@@ -135,10 +156,12 @@ annotation to your .proto file
 5. Generate reverse-proxy
 
    ```sh
-   protoc -I/usr/local/include -I. \
+   protoc -I/usr/local/include \
+     -I${ROOT_TO_PROTO} \
      -I$GOPATH/src \
      -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
-     --grpc-gateway_out=logtostderr=true:. \
+     --plugin=proto-gen-grpc-gateway=${GOPATH}/bin/protoc-gen-grpc-gateway \
```

Comment:
Specifying the `plugin` is not necessary if it is in your path already. Using `go get` should put it in your path. I don't think we need to add this.